src/pages/Videos.js

```diff
@@ -41,7 +42,18 @@ const Videos = () => {
 
                 <VideosFilter />
 
-                <VideosSection />
+                {
+                    type === 'error'
+                        ? <Alert classes='bg-err'>{message}</Alert>
+                        : type === 'success' ? <Alert classes='bg-success'>{message}</Alert>
+                            : ''
+                }
+
+                {
+                    loading
+                        ? <Text classes={`${getTextColor(theme)} txt-xlg txt-500 txt-cap mg-top-md`}>loading...</Text>
```

Comment:
you can also use loader package for loading here

Reply:
I'll look into it definitely.